src/core/components/model.jsx

```diff
@@ -224,6 +224,7 @@ class Model extends Component {
 
     if ( schema && (schema.get("type") || schema.get("properties")) ) {
       modelSchema = schema
+      modelName = name
```

Comment:
Unfortunately I'm not a frontend programmer and this style looks ugly for me. :)
But real problem is when definition looks like 
```js
{
  // ...
  "SomeDefinititionName": {
    "allOf": [
      {
        "$ref": "<SOME_REF>"
      },
      // ...
    ]
  },
  // ...
}
```

`schema.get("$$ref")` returns wrong value